Changelog.md

```diff
@@ -5,10 +5,11 @@ Language Features:
 
 Compiler Features:
  * Commandline Interface: Normalize paths specified on the command line and make them relative for files located inside base path.
+ * Commandline Interface: Disallowed the ``--experimental-via-ir`` option to be used with Standard Json, Assembler and Linker modes.
```

Comment:
Can we avoid this practice of re-sorting other changelog entries as part of PRs?

Reply:
It's always nice to `git blame` the changelog and find out which PR introduced a feature...

Reply:
Sure, just wanted to avoid the Changelog looking messy already, since there are still only a few lines

Reply:
Fixed